BLSQ/iaso
# Wc2-115: make storage device.status updated at mandatory at the model level

The StorageDevice.status_updated_at field is now mandatory at the Model level, for robustness reasons (see WC2-115)

We also make sure that when a device is created, it is set to the current time (because the status is created in the OK status, we can conceptually consider this as an initial status "change")

## Self proof reading checklist


## How to test

- Make sure storage devices can still be created/edited in the admin
- Make sure that when a new device is created, its `status_updated_at` field is set to the current time.
- Make sure the Storage/Storage log screens still work in the UI
- Check interactions with the mobile app (that's for @bmonjoie)

## Notes

The admin screen has also been improved for StorageDevices (missing fields that made difficult to test/debug the feature)
